examples/01-filter/contouring.py

```diff
@@ -84,8 +84,6 @@
 #
 # Create labeled surfaces from 3D label maps (e.f. multi-label image segmentation)
 # using :func:`contour_labeled() <pyvista.ImageDataFilters.contour_labeled>`.
-# Requires VTK version 9.3
-if pv.vtk_version_info >= (9, 3):
-    label_map = pv.examples.download_frog_tissue()
-    mesh = label_map.contour_labeled()
-    mesh.plot(cmap="glasbey_warm", cpos="yx", show_scalar_bar=False)
+label_map = pv.examples.load_frog_tissue()
+mesh = label_map.contour_labeled(smoothing=True)
+mesh.plot(cmap="glasbey", cpos="yx", show_scalar_bar=False, clim=mesh.get_data_range())
```

Comment:
This is still necessary because there are cases where users run vtk versions earlier than v9.3.
```suggestion
if pv.vtk_version_info >= (9, 3):
    label_map = pv.examples.load_frog_tissue()
    mesh = label_map.contour_labeled(smoothing=True)
    mesh.plot(cmap="glasbey", cpos="yx", show_scalar_bar=False, clim=mesh.get_data_range())
```

Reply:
Sounds good. This was the first time I had seem something like this in the docs. 

So this line (from a separate example in the docs) is considered a bug or bad practice since it can only execute with  vtk >= 9.1?
https://github.com/pyvista/pyvista/blob/0007ba4c536b5b795e85a381595def5238bf522c/examples/99-advanced/openfoam-tubes.py#L69

Reply:
Yes, it would be helpful to have a separate PR for that one.